src/containers/SocialLinksGroup.jsx

```diff
@@ -0,0 +1,25 @@
+import SocialLink from "../components/SocialLink";
+
+const SocialLinksGroup = () => {
+  return (
+    <div className="flex justify-center">
```

Comment:
Add `gap-2` as in previous file I had removed `m-2`

Reply:
Added gap to parent as per review comments